### DISCUSSION TITLE
Recurting

### DISCUSSION THREAD

**Discussion by raingggg** [raingggg/coctohug#5] [Announcements]
CoctoHug invites you join the community: 
1. One nodejs developer - parttime is fine 
2. Multiple active community ambassador - xch as reward
Join [Discord](https://discord.gg/umfKHm7gVM) and leave me a message. Let's discuss more about the project plan.

**Comment by lotkongd** [comment]
I'm a student. Can I join?

**Reply by raingggg** [reply]
Welcome! Let's discuss on Discord!
